internal/http/services/owncloud/ocdav/propfind.go

```diff
@@ -384,14 +385,35 @@ func (s *svc) mdToPropResponse(ctx context.Context, pf *propfindXML, md *provide
 		lastModifiedString := t.Format(time.RFC1123Z)
 		response.Propstat[0].Prop = append(response.Propstat[0].Prop, s.newProp("d:getlastmodified", lastModifiedString))
 
+		// stay bug compatible with oc10, see https://github.com/owncloud/core/pull/38304#issuecomment-762185241
+		var checksums strings.Builder
```

Comment:
wouldn't it actually be easier by using a struct with XML tags and [xml.Marshal](https://golang.org/pkg/encoding/xml/#Marshal)?

Reply:
it would if we were rendering the whole propfind using an xml marshaler. Now we would be assembling a new property, running the xml marshaler on it, and assigning the bytes to the prop ... something for a different PR.